spec/slack-ruby-bot/commands/unknown_spec.rb

```diff
@@ -4,9 +4,16 @@
   def app
     SlackRubyBot::App.new
   end
+
+  def unknown_response
```

Comment:
Run `rubocop --auto-gen-config` to update that number.

On Fri, Nov 20, 2015 at 11:56 AM, Cody Rayment <notifications@github.com>
wrote:

> In spec/slack-ruby-bot/commands/unknown_spec.rb
> <https://github.com/dblock/slack-ruby-bot/pull/29#discussion_r45490821>:
>
> > @@ -4,9 +4,16 @@
> >    def app
> >      SlackRubyBot::App.new
> >    end
> > +
> > +  def unknown_response
>
> The reason I pulled this out was because RuboCop was failing for the line
> being to long. I have restored the failure behaviour so you can see. This
> was also noted in the commit message but now they are squashed (strange to
> me). Let me know how you would like to resolve the line being too long -
> tweak RuboCop settings? Extract in another way?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dblock/slack-ruby-bot/pull/29/files#r45490821>.
>



-- 

dB. | Moscow - Geneva - Seattle - New York
code.dblock.org - @dblockdotorg <http://twitter.com/#!/dblockdotorg> -
artsy.net - github/dblock <https://github.com/dblock>